docs/user_guide/vis/index.rst

```diff
@@ -1,88 +1,285 @@
+.. _IFV:
 
-Visualization
-=============
+IDAES Flowsheet Visualizer
+===========================
 
-.. warning::
-    The visualization tool is still in active development and we
-    hope to improve on it in future releases. Please use its
-    functionality at your own discretion.
+.. contents::
+    :depth: 2
 
-Overview
+Concepts
 --------
+The IDAES Flowsheet Visualizer, or IFV for short, is a graphical user interface for viewing an IDAES flowsheet.
+It uses the visual language of traditional process engineering diagrams to display
+the components of a given flowsheet,
+their connections, and values associated with unit models and streams. While the changes made from the UI do
+not affect the underlying model, changes in the flowsheet -- e.g., from Python code or from solvers updating the
+stream values -- are immediately available to be displayed in the IFV. The IFV does not require running a
+separate program, as its functionality is embedded in the invoking Python code (see the `Client/server architecture <ifv-architecture>`_ for details).
+Starting and stopping the IFV is fast and does not consume many resources, a fact which we hope encourages
+its frequent use.
+
+Quickstart
+----------
+This section will walk you through constructing a basic flowsheet and visualizing it with the IFV.
+
+Before starting to use the IFV, you must have installed IDAES. For detailed instructions on how to do this,
+see :ref:`IDAES Installation`.
+
+In this quickstart we will use a Jupyter Notebook to compose our flowsheet and view it with the IFV. So, first
+you need to open a new, blank, Jupyter Notebook. You can start the Jupyter Notebook (server) from the terminal command-line with
+the command ``jupyter notebook``. See the `official Jupyter Notebook documentation <https://jupyter-notebook.readthedocs.io/>`_
+for detailed instructions. The Jupyter Notebook is a web application, so you should get a web page (often as a tab
+on an open browser window) that shows the contents of the directory from where you issued the command. Navigate to
+a directory where you want to save your work, then click on the "New" button in the upper-right corner to create
+a new notebook. You may have multiple options under the "Python" sub-menu; make sure you select a Python installation
+that has IDAES already installed.
+
+We need to create, in this Jupyter Notebook, an IDAES flowsheet to visualize. Paste the following code (taken from the Flash unit tutorial)
```

Comment:
Emphasize this is an example code.

As an example, lets use the `Flash Unit Tutorial`, in which we import the idaes and pyomo libraries, create the Pyomo model, create the idaes flowsheet, build a property package, and add a unit model (flash separation) to the flowsheet. Then, we set the inlet conditions of the flash by fixing the inlet flowrate, temperature, pressure, and molar fraction. Also, fix the heat duty and pressure drop to 0. Finally, initialize the flowsheet.

Reply:
How about I link to the full tutorial? The details of what it is doing really are not relevant to this example -- the idea is just to show that something can be visualized.

Reply:
I added a link to the tutorials page.